api_v3/lib/types/partner/KalturaPartner.php

```diff
@@ -469,6 +469,12 @@ class KalturaPartner extends KalturaObject implements IFilterable
 	 */
 	public $eventPlatformAllowedTemplates;
 	
+	/**
+	 * @var string
+	 * @readonly
+	 */
+	public $verticalClassificationName;
```

Comment:
didnt we agreed that this will be a number?
why the name contisn 'Name' and why is it a string?